package.json

```diff
@@ -39,17 +64,12 @@
     "tslint": "tslint -p . -c tslint.json",
     "tsc": "tsc",
     "prebuild": "rimraf dist && rimraf typings",
-    "build": "npm run tslint && npm run tsc",
+    "build": "npm run tslint && tsc -p . && rollup -c",
+    "build-browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c",
+    "build-node": "tsc -p . && cross-env ONLY_NODE=true rollup -c",
+    "build-test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js",
     "test": "npm run build",
-    "unit": "nyc --reporter=lcov --reporter=text-lcov mocha -r ts-node/register -t 50000 test/**/*.spec.ts",
+    "unit": "npm run build-test && mocha -t 50000 test-dist/index.js",
     "prepack": "npm i && npm run build"
-  },
-  "homepage": "http://github.com/azure/azure-servcie-bus-node",
```

Comment:
Broken Link : https://github.com/azure/azure-servcie-bus-node
Change it to https://github.com/azure/azure-service-bus-node
Same thing at the lines 50 and 53 :)

Reply:
Great catch!!! I'll change that!